main.gms

```diff
@@ -855,6 +852,16 @@ parameter
 ;
   cm_gdximport_target      = 0;      !! def = 0
 *'
+parameter
```

Comment:
I fully agree with cm_33gs_ew, cm_33LimRock, as they're only used in the CDR module. However, I will change it in a separate PR.  When it comes to cm_frac_CCS, cm_frac_NetNegEmi, c_ccsinjecratescen, c_ccscapratescen, since these are used in the core, I'd keep them as they are, what do you think?